src/nix/daemon.cc

```diff
@@ -555,7 +565,7 @@ struct CmdDaemon : StoreCommand
 
     void run(ref<Store> store) override
     {
-        runDaemon(stdio, isTrustedOpt);
+        runDaemon(stdio, isTrustedOpt, false);
```

Comment:
It would be good to add the flag to the new CLI too. And let's make it experimental under `Xp::MountedSSHStore` too.

Reply:
I totally missed the fact that the new CLI is in the same file. Added it all now.